functions.php

```diff
@@ -503,7 +504,7 @@ function tlpd_run_once() {
 		/**
 		 * Most events will be in British Columbia
 		 */
-		update_option( 'tlpd_location_default_province', 'British Columbia' );
```

Comment:
by removing `update_option( 'tlpd_location_default_province', 'British Columbia' );`, we can expect an empty value at line 139 in `plugins/events-manager/forms/event/location.php`

https://github.com/BCcampus/tlpd/blob/dev/plugins/events-manager/forms/event/location.php#L137

Reply:
Nice catch, will include both so line 139 doesn't have an empty value